### PR TITLE
Silicon alu autoclave update

### PR DESCRIPTION
**Pull request by Dream-Master**
add 4 slots to Autoclave
change silicon and alu recipes
* https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/209
* https://github.com/GTNewHorizons/bartworks/pull/45
* https://github.com/GTNewHorizons/GTplusplus/pull/29
* https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/8623

### CODE REVIEW

**Comment by Prometheus0000** [comment]
What's the reason to add a bunch of slots to the autoclave? I don't think it needed any more?

And what's the reason for changing silicon? And aluminium?

**Comment by Dream-Master** [comment]
because it was boaring todo and far away from rl. Now it makes the game more interreting. Sure it needs tweaking

**Comment by GlodBlock** [comment]
it is strange that you have to turn aluminum dust into alumine before smelting. it is that alumine should be turned into aluminum dust to smelt IRL

**Comment by Dream-Master** [comment]
it is but remove all alu dust of all centrifuge recipe and electrolyse recipes would a pain or you have an idea how replace this with maybe some base aluminium material ?

File: src/main/java/gregtech/api/enums/Materials.java [hunk 1]
```diff
@@ -769,10 +779,10 @@
...
-    public static Materials ElectricalSteel         = new Materials( 365, TextureSet.SET_METALLIC          ,   6.0F,    512,  2, 1|2          |64|128      , 185, 185, 185,   0,   "ElectricalSteel"         ,   "Electrical Steel"              ,    0,       0,       1811, 1000,  true, false,   4,   1,   1, Dyes.dyeGray        , 1, Arrays.asList(new MaterialStack(Steel, 1), new MaterialStack(Coal, 1), new MaterialStack(Silicon, 1))).disableAutoGeneratedBlastFurnaceRecipes();
+    public static Materials ElectricalSteel         = new Materials( 365, TextureSet.SET_METALLIC          ,   6.0F,    512,  2, 1|2          |64|128      , 185, 185, 185,   0,   "ElectricalSteel"         ,   "Electrical Steel"              ,    0,       0,       1811, 1000,  true, false,   4,   1,   1, Dyes.dyeGray        , 1, Arrays.asList(new MaterialStack(Steel, 1), new MaterialStack(Coal, 1), new MaterialStack(SiliconDioxide, 1))).disableAutoGeneratedBlastFurnaceRecipes();
```
Comment (GlodBlock) [inline src/main/java/gregtech/api/enums/Materials.java:785]: here the allay shouldn't change

File: src/main/java/gregtech/api/enums/Materials.java [hunk 2]
```diff
@@ -769,10 +779,10 @@
...
 
     //\/HAD TO MOVE DOWN SECTION
-    public static Materials RedstoneAlloy           = new Materials( 381, TextureSet.SET_METALLIC          ,   3.0F,    128,  2, 1|2          |64|128      , 181,  51,  51,   0,   "RedstoneAlloy"           ,   "Redstone Alloy"                ,    0,       0,        671, 1000,  true, false,   1,   1,   1, Dyes.dyeRed         , 1, Arrays.asList(new MaterialStack(Redstone, 1), new MaterialStack(Silicon, 1), new MaterialStack(Coal, 1))).disableAutoGeneratedBlastFurnaceRecipes();
+    public static Materials RedstoneAlloy           = new Materials( 381, TextureSet.SET_METALLIC          ,   3.0F,    128,  2, 1|2          |64|128      , 181,  51,  51,   0,   "RedstoneAlloy"           ,   "Redstone Alloy"                ,    0,       0,        671, 1000,  true, false,   1,   1,   1, Dyes.dyeRed         , 1, Arrays.asList(new MaterialStack(Redstone, 1), new MaterialStack(SiliconDioxide, 1), new MaterialStack(Coal, 1))).disableAutoGeneratedBlastFurnaceRecipes();
```
Comment (GlodBlock) [inline src/main/java/gregtech/api/enums/Materials.java:782]: here the allay shouldn't change

Comment (Dream-Master) [reply src/main/java/gregtech/api/enums/Materials.java]: fixed

File: src/main/java/gregtech/api/enums/Materials.java (hunk 1 above)
Comment (Dream-Master) [reply src/main/java/gregtech/api/enums/Materials.java]: fixed

**Pull request merged**